src/Config/ConfigFactory.php

```diff
@@ -4,16 +4,27 @@
 
 namespace Shopware\Deployment\Config;
 
-use Symfony\Component\Filesystem\Path;
+use Symfony\Component\Finder\Finder;
 use Symfony\Component\Yaml\Yaml;
 
 class ConfigFactory
 {
     public static function create(string $projectDir): ProjectConfiguration
     {
-        $file = Path::join($projectDir, '.shopware-project.yml');
+        $finder = new Finder();
```

Comment:
what about adding `.shopware-project.yaml` as fallback file. starting up a Finder seems to be a bit overkill :grin: 

also extending the test `\Shopware\Deployment\Tests\Config\ConfigFactoryTest` would be nice

Reply:
overkill never good :smile: